database/javascript/2017/16175.yaml

```diff
@@ -0,0 +1,28 @@
+---
+cve: 2017-16175
+title: CVE in ewgaddis.lab6
+description: >
+    ewgaddis.lab6 is a file server. ewgaddis.lab6 is vulnerable to a directory traversal issue, giving an attacker access to the filesystem by placing "../" in the url.
+cvss_v2: 5.0
+references:
+    - https://github.com/JacksonGL/NPM-Vuln-PoC/blob/master/directory-traversal/ewgaddis.lab6
+    - https://nodesecurity.io/advisories/465
+
+affected:
+    - name: ewgaddis.lab6
+      version:
```

Comment:
https://nvd.nist.gov/vuln/detail/CVE-2017-16175?cpeVersion=2.2#vulnConfigurationsArea

Reply:
+1 please update the PR